ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -73,7 +76,7 @@ - (id)init
     }
     
     _validatedAdfsAuthorities = [NSMutableDictionary new];
-    _aadCache = [ADAadAuthorityCache new];
+    _aadCache = [MSIDAadAuthorityCache sharedInstance];
```

Comment:
If it's a singleton, it doesn't need to be an instance variable

Reply:
you are right!